apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -121,6 +121,28 @@ export function InstructorCourseAdminInstances({
                 </tbody>
               </table>
             </div>
+            ${courseInstances.length === 0
+              ? html`
+                  <div class="my-4">
+                    <p class="text-center font-weight-bold">No course instances found.</p>
+                    <p class="text-center mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
```

Comment:
cc @mfsilva22 to review the copy here.

Reply:
Not yet requesting a change, just some food for thought when Mariana reviews: I'm wondering if it's better to try to describe what a course instance actually is rather than some example names. For instance, "A course instance contains the assessments and other configuration for a single offering of a course."

Reply:
I like the suggestion by @nwalters512 above, but I think we should still offer an example (works as "best practice" in this case). I would just use one ("Fall 2024"). 

Based on my experience running queries, sometimes instructors use course instance names that are not helpful. Some include the name of the course, not knowing that the course name and course instance name get both appended when displayed in the "Courses" list

Reply:
I think we can suggest best practices in #11004 so that the suggestion is visible directly next to the input where they'll actually pick a name. Does that sound reasonable?

Reply:
yes, I Iike that!

Reply:
Looks great!